### PR TITLE
Register genericx.is-a.dev

### PR DESCRIPTION
**Pull request by xgeneric**
Register genericx.is-a.dev with CNAME record pointing to paulswebhelp.com.

(disregard the "rh" from earlier)

### CODE REVIEW

File: domains/xgeneric.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "xgeneric",
+           "discord": "1194494419064328296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.D232zLsD29QIoCDQBNjD76jn7VgIPpSOIrRPBMn37VV_JTS_3fJAKK7ADQtFwQu0A-I0TcZTU3UczrEGdwjmkHHAhPu95BuRbFqbWruAnX6ETw0JjSJQvYmlrFjF1xDMM33CJ3GW0R985HyTcB5MbU6-QvWbqPou5bwRShlQ06KKP944EiASahDDNAog2HUwEf7VJjWxdB-hP3Yj_BOvlC_kQBfZBhOTG4OWwbvbAonKjK26oqrVdUKhuDT0VfvI4VsBXlSG3g8P2LnyZZPmh3xJ3UQ4aWYRFhJkC3f0M98NQ4mI7ddwZu4YPVcw9yHiMD9iDnTBnNiSsJVZeMhqQw.6v7QP1iKQEXF2xIliVLoxA.n1gPGIhivHdAaq9enhjewqRKzW2BtPsOt97Ng4x7yp8nv-OzAS4Jev1hp_M3z0XL7l5z1Wc2YrPW7z3trdUlmTZkCLcOziKhwwECaKbQNErOq3yXDxe7K5FvhmQWGNQJ.fZiP_f8z_xpVs5jS0MRQPQ"
+        },
+    
+        "record": {
+            "CNAME": "paulswebhelp.com"
```
Comment (DEV-DIBSTER) [inline domains/xgeneric.json:9]: Unable to access site. Provide a preview.

File: domains/genericx.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "xgeneric",
+           "discord": "1194494419064328296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.DNr2jxeR-0YVkAChAl4uPX31TrkGI4S8REI18sZvz4gGrL9c2LRtnnHlJfq0fMntFjAvLoD2lvb-kbjDq-INycXI0YHr-MG4YIvJmccyAmaxmtfhIqSWQc6EwMRar7mx0aE3NaOXJSKZQSpH86MVHqO4X9TZMGxkCfq9IAaXHmAVBazCNPFySzCY2jHo3LRmDFsJyTf0mWm_RwaYuPlS7__slgs2XQnWCYNP_kdMwENR7dDFVG9IjwSXQI-D12nyqquN_oM6WDg2ajfwMdbfd3aOpNkLOvAj-slpw81v2zccP040MrQJdTlN_LiMGFMqUPqNqev3lDwu-up8xDKIzQ.Fq8v4wlDvYQWahwdgcSXPQ.b_b4xs8S2pJNAPSPFJfZ6u5nMlNfAnxnm0UbH_LrqXCGIMX5ymm54wsnWFrymwp9OSqTbotZeYMFXWZlCouMxXOqrOmLHb58OJmsRZarQ0lqSOduCvFnonngL6gTpa1s.1o8XkyzrR4qKrVrinRJhzQ"
+        },
+    
+        "record": {
+            "CNAME": "paulswebhelp.com"
```
Comment (DEV-DIBSTER) [inline domains/genericx.json:9]: Unable to access site. Provide a preview.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/genericx.json (hunk 2 above)
Comment (xgeneric) [reply domains/genericx.json:9]: Here is a preview of the website: 
image

File: domains/xgeneric.json (hunk 1 above)
Comment (xgeneric) [reply domains/xgeneric.json:9]: Here is a preview: 
image

**Pull request closed without merging**